client/js/app/components/explorer/query_builder/extraction_properties_filter.js

```diff
@@ -0,0 +1,45 @@
+var React = require('react');
```

Comment:
I'm definitely up for moving to es2017 syntax. It hurt me emotionally to write `var`.